app/helpers/application_helper.rb

```diff
@@ -7,4 +7,8 @@ def order_articles_options
      ['Sort by Description', 'sort_by_description_length'],
      ['Sort by User email', 'sort_by_users_email']]
   end
+
+  def users_for_articles
+    User.where('id <> ?', current_user.id).pluck(:nickname, :id)
```

Comment:
chenge to 
`def users_for_articles(user_ids=[])`
`User.where.not(id: user_ids).pluck(:nickname, :id)`